electron/ironfish/AssetManager.ts

```diff
@@ -0,0 +1,70 @@
+import { Asset as NativeAsset } from '@ironfish/rust-nodejs'
+import { AssetValue } from '@ironfish/sdk/build/src/blockchain/database/assetValue'
+import { GENESIS_BLOCK_PREVIOUS } from '@ironfish/sdk/build/src/primitives/block'
+import Asset from 'Types/Asset'
+import IIronfishAssetManager from 'Types/IronfishManager/IIronfishAssetManager'
+import AbstractManager from './AbstractManager'
+
+class AssetManager extends AbstractManager implements IIronfishAssetManager {
+  async list(search?: string, offset = 0, max = 100): Promise<Asset[]> {
+    const nativeAsset = await this.default()
+    const assets: Asset[] = []
+    const isApproach = (asset: Asset) =>
```

Comment:
question: Just curious, what does `isApproach` mean here?

Reply:
It my mistake. Fixed.